src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFMerger.java

```diff
@@ -129,14 +132,26 @@ private void addDocument(SortableBundleItem item) throws IOException {
                         currentPageNumber + newDoc.getNumberOfPages());
             }
 
-            if (tableOfContents != null) {
+            if (tableOfContents != null && newDocOutline!= null) {
+                PDOutlineItem anySubtitlesForItem =newDocOutline.getFirstChild();
+                while(anySubtitlesForItem != null){
+                    siblings.add(anySubtitlesForItem);
+                    anySubtitlesForItem= anySubtitlesForItem.getNextSibling();
+                }
                 tableOfContents.addDocument(item.getTitle(), currentPageNumber, newDoc.getNumberOfPages());
+                for (PDOutlineItem subtitle:siblings){
+                    tableOfContents.addDocumentWithOutline(item.getTitle(), currentPageNumber, newDoc.getNumberOfPages(),subtitle);
```

Comment:
formatting issue, whitespace after commas and variables

Reply:
done